proto/cellar_v2.proto

```diff
@@ -373,5 +374,7 @@ message AdaptorCall {
         CellarAdaptorV1Calls cellar_v1_calls = 15;
         // Represents function calls to the UniswapV3Adaptor V2
         UniswapV3AdaptorV2Calls uniswap_v3_v2_calls = 16;
+        // Represents function calls to the AaveV2EnableAssetAsCollatorAdaptor V1
+        AaveV2EnableAssetAsCollateralAdaptorV1Calls aave_v2_enable_asset_as_collateral_v1_calls = 17;
```

Comment:
Looks like we're adding this to the 2.0 proto but have not done 2.2.

Reply:
AdaptorCall is a top level type in the proto, this applies to both. It's split out in the handler files.

Reply:
Ah, I see that now, 2.2 is split for some files but not here, roger that.

Reply:
Though the changes you've made support this adaptor for 2.0 as well, wouldn't we need to add it to the Cellar V2.0 section of this file?

Reply:
Never mind, I see it now.